packages/migrate/test/__snapshots__/migrate.test.ts.snap

```diff
@@ -185,6 +263,41 @@ export default class TestGjsNoErrors extends Component {
 "
 `;
 
+exports[`fix > .gts > strips jsdoc param with missing name 1`] = `
+"import Component from \\"@glimmer/component\\";
+
+export class Something extends Component {
+  /** */
```

Comment:
This is happening because of `deleteUnmatchedParameter`

Reply:
Ah, yes. I saw that in our tests but didn't know what they were for. We have tests covering ts-codefixes so we can validate the changes are operating as we expect them to.